source/gke.rst

```diff
@@ -176,11 +176,11 @@ This command will connect you to the MySQL monitor.
 
    mysql: [Warning] Using a password on the command line interface can be insecure.
    Welcome to the MySQL monitor.  Commands end with ; or \g.
-   Your MySQL connection id is 111
-   Server version: 5.5.30 (ProxySQL)
+   Your MySQL connection id is 119
+   Server version: 8.0.19 (ProxySQL)
```

Comment:
should we rework it for HAProxy?

Reply:
I think it's better to have at least one ProxySQL example somewhere

Reply:
done